packages/eslint-plugin/src/__tests__/unsupported-route-components.test.ts

```diff
@@ -38,12 +38,32 @@ ruleTester.run('unsupported-route-components', unsupportedRouteComponents, {
           )
         }`.replace(/ +/g, ' '),
     },
+    {
+      code: `
+        const AnotherThing = <Bazinga><p>Hello</p></Bazinga>
+        const Routes = () => {
+          return (
+            <Router>
+              <PrivateSet
+              whileLoadingAuth={AnotherThing}
+              >
```

Comment:
Sorry about the nitpicking here, but could you put all of this on one line?

Reply:
![image](https://github.com/redwoodjs/redwood/assets/30793/187e843e-6fad-41b5-9a67-624a60c0efdf)
 😄  This made me smile